raster/r.random/support.c

```diff
@@ -5,7 +5,7 @@
 #include "local_proto.h"
 
 
-int make_support(struct rr_state *theState, int percent, double percentage)
+int make_support(struct rr_state *theState, int percent, double percentage, long seed_value)
```

Comment:
long is not portable: long is 32 bit on 32 bit systems and 64 bit on 64 bit systems, on MS Windows it is always 32 bit. Use grass_int64 if you need a portable 64 bit signed integer type.